packages/app-mobile/components/NoteEditor/NoteEditor.tsx

```diff
@@ -31,143 +32,142 @@ function fontFamilyFromSettings() {
 	return [f, 'sans-serif'].join(', ');
 }
 
-// function useCss(themeId:number):string {
```

Comment:
Please don't comment out the old code because then we can't know what has changed from the diff.

Reply:
Thanks, but could you put this commit in this pull request? Basically there shouldn't be any commented out code in the PR as it prevents reviewing it.

Reply:
I'm confused... The commit should be part of this pull request.

Reply:
![Highlighted minus](https://user-images.githubusercontent.com/46334387/159698338-b0c8f3ff-bace-463c-a126-b6213d92cbce.jpeg)
Please note that this is a `-` and not a `+`.

Reply:
See https://github.com/laurent22/joplin/issues/6316